packages/types/src/feedback/config.ts

```diff
@@ -84,7 +84,7 @@ export interface FeedbackTextConfiguration {
   /**
    * The label for the Feedback widget button that opens the dialog
    */
-  buttonLabel: string;
+  buttonLabel: string | { desktop: string;  mobile: string};
```

Comment:
We can allow `undefined` in here if the props for Actor change from `buttonLabel: string` to `buttonLabel: string | undefined`

```suggestion
  buttonLabel: string | { desktop?: string | undefined;  mobile?: string | undefined};
```